core/types/src/svn.rs

```diff
@@ -39,3 +42,25 @@ pub struct CpuSvn(sgx_cpu_svn_t);
 impl_newtype_for_bytestruct! {
     CpuSvn, sgx_cpu_svn_t, SGX_CPUSVN_SIZE, svn;
 }
+
+/// Config ID
```

Comment:
Not fond of the config id landing in `svn.rs`.  Looking at the sgx c code it looks like it's in `sgx_key.h`, it seems that it's a type without a clear home.